src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp

```diff
@@ -239,9 +239,9 @@ namespace {
         for (const auto& record : handlerContext.keyword) {
             std::fill(max.begin(), max.end(), record.getItem("DRSDT_MAX").getSIDouble(0));
             std::fill(options.begin(), options.end(), record.getItem("Option").get< std::string >(0));
-            OilVaporizationProperties ovp = this->m_oilvaporizationproperties.get(handlerContext.currentStep);
+            OilVaporizationProperties ovp = handlerContext.sched_step.oilvap();
             OilVaporizationProperties::updateDRSDT(ovp, max, options);
-            this->m_oilvaporizationproperties.update( handlerContext.currentStep, ovp );
+            handlerContext.sched_step.oilvap( ovp );
```

Comment:
Is there a reason we're not using `this->snapshots.back()` here?

Reply:
Rebase rebase .... 😟